types/mailgun-js/index.d.ts

```diff
@@ -74,6 +75,11 @@ declare namespace Mailgun {
             "recipient-variables"?: BatchSendRecipientVars;
         }
 
+        interface SendTemplateData extends SendData {
```

Comment:
Let's just avoid using `any` by defining `SendTemplateData` as
```typescript
type SendTemplateData = SendData & {
    template: string;
    [templateVariable: string]: string;
}
```

Then I am happy to approve :)

Reply:
@andipaetzold Sorry for the delay, I've been really busy with other things.  I've implemented this fix.